pkg/cmd/authTokensGet.go

```diff
@@ -134,7 +136,12 @@ func (cmd *AuthTokensGetCommand) executeAuthTokensGet(
 
 				if err == nil {
 					// Call to process the command in a unit-testable way.
-					err = auth.GetTokens(apiClient, console)
+					// Checking if the loginId param was passed
+					if cmd.cobraCommand.Flags().Changed("id") {
+						err = auth.GetTokensByLoginId(apiClient, console, authTokenCmdValues.name)
+					} else {
+						err = auth.GetTokens(apiClient, console)
+					}
```

Comment:
Instead of checking if the flag's value has changed, we can pass the loginId value into the `GetTokens` function and then handle it in there - see `runs get` for reference.